README.md

```diff
@@ -125,6 +125,19 @@ fetch('http://httpbin.org/post', { method: 'POST', body: stream })
 		console.log(json);
 	});
 
+// post with JavaScript object
+
+var body = { a: 1 };
+fetch('http://httpbin.org/post', { 
+        method: 'POST',
+        body:    JSON.stringify(body),
+        headers: { 'Content-Type': 'application/json' },
+    }).then(function(res) {
+        return res.json();
+    }).then(function(json) {
+        console.log(json);
+    });
```

Comment:
Hadn't noticed that, I never use tabs! Should I commit a revision which can be squashed when PR is merged, or should I squash commits (or just scratch it and start over)?